editor/gui/editor_title_bar.cpp

```diff
@@ -80,6 +80,47 @@ void EditorTitleBar::gui_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
+void EditorTitleBar::_notification(int p_what) {
+	if (p_what == NOTIFICATION_SORT_CHILDREN) {
```

Comment:
This technically could be an early return, although maybe it's not a good idea in _notification 🤔

Reply:
Since it's a single if, it makes sense to make it early return.